src/ASM/IntegrandBase.h

```diff
@@ -359,6 +370,14 @@ class NormBase : public Integrand
                            const std::vector<size_t>& elem_sizes,
                            const std::vector<size_t>& basis_sizes,
                            LocalIntegral& elmInt);
+  //! \brief Initializes current element for numerical integration (mixed).
+  virtual bool initElement(const std::vector<int>& MNPC,
+                           const MxFiniteElement& fe,
```

Comment:
`fe` is unused (will give compiler warning?)

Reply:
made anonymous.